src/db/tables/onerep_scans.ts

```diff
@@ -397,6 +398,42 @@ async function getEmailForProfile(onerepProfileId: number) {
   }
 }
 
+async function getScanResultsWithBrokerUnderMaintenance(
+  onerepProfileId: number | null,
+) {
+  if (onerepProfileId === null) {
+    return null;
+  }
+
+  let scanResults = await knex("onerep_scan_results as sr")
+    .select(
+      "sr.*",
+      "s.*",
+      "sr.status as scan_result_status", // rename to avoid collision
+      "db.status as broker_status", // rename to avoid collision
+    )
+    .innerJoin("onerep_scans as s", "sr.onerep_scan_id", "s.onerep_scan_id")
+    .where("s.onerep_profile_id", onerepProfileId)
+    .andWhere("sr.manually_resolved", "false")
+    .andWhereNot("sr.status", "removed")
+    .join("onerep_data_brokers as db", "sr.data_broker", "db.data_broker")
+    .orderBy("sr.onerep_scan_result_id");
+
+  console.log("\n\n scan results broker maintenance:");
+  console.log(scanResults.length);
+
+  scanResults = scanResults.filter(
+    (result) =>
+      result.broker_status === "removal_under_maintenance" ||
+      new Date().getTime() - new Date(result.updated_at).getTime() >
+        200 * 24 * 60 * 60 * 1000,
+  );
+  console.log({ scanResults });
+  console.log(scanResults.length);
```

Comment:
Same here re: `logger`, not sure if these are intended to keep or just for debugging.

Reply:
```suggestion
```